packages/frontend/app/styles/components/sidebar.css

```diff
@@ -0,0 +1,19 @@
+div.contact-grid {
```

Comment:
are all these classes needed? 
maybe just apply the `grid` class to the container?, which is defined as:
```
.grid {
  @apply --grid;
}
```

Reply:
as is, the pin button is a big large
![image](https://user-images.githubusercontent.com/199018/68306052-cb967e80-0076-11ea-990a-77f6093f975d.png)

Reply:
playing around with this, it looks like if the grid-areas go away, and the containing div be given:
```css
@apply --column-grid

```

Reply:
That is a lot nicer and cleaner!